test/manual/amp-vine.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script custom-element="amp-vine" src="../../dist/v0/amp-vine-0.1.max.js" async development></script>
+  <script src="../../dist/amp.js" development></script>
```

Comment:
can you pleaese move this as the last child of `head` and add the `async` attribute. Thanks!